### PR TITLE
i10L integration

### PR DESCRIPTION
**Pull request by aruznieto**

### CODE REVIEW

File: chameleonultragui/lib/helpers/general.dart [hunk 1]
```diff
@@ -131,15 +134,15 @@ Color hexToColor(String hex) {
 
 String platformToPath() {
   if (Platform.isAndroid) {
-    return "android";
+    return "Android";
```
Comment (Foxushka) [inline chameleonultragui/lib/helpers/general.dart:137]: This is used for tests workaround, not displayed to user. Revert changes

Comment (aruznieto) [reply chameleonultragui/lib/helpers/general.dart]: Done

**Comment by Foxushka** [comment]
image
Is it todo (in slot manager dump loading) or it is correct translation?

**Comment by Foxushka** [comment]
I see
image

**Comment by aruznieto** [comment]
> Is it todo (in slot manager dump loading) or it is correct translation?

"Todo" in Spanish means "All"

**Review by Foxushka — APPROVED** [review]
Looking nice to me. Thank you for your work

**Comment by GameTec-live** [comment]
2 things: Maybe add a link to crowding in the readme
and, shouldnt there be a flag symbol or did i missread smthng?
image

**Review by GameTec-live — APPROVED** [review]

**Comment by aruznieto** [comment]
> Maybe add a link to crowding in the readme
> and, shouldnt there be a flag symbol or did i missread smthng?

I change the symbol because there are a lot of countries that speak Spanish and only to put the flag of one of them... I think is not correct, but If you want, I can change it.

I´m going to add the link to crowdin but, in what project? Mine or fox?

**Comment by GameTec-live** [comment]
> > Maybe add a link to crowding in the readme
> > and, shouldnt there be a flag symbol or did i missread smthng?
> 
> I change the symbol because there are a lot of countries that speak Spanish and only to put the flag of one of them... I think is not correct, but If you want, I can change it.
> 
> I´m going to add the link to crowdin but, in what project? Mine or fox?

id say your project, because thats where you alrdy translated some stuff, so we got some full progess bars... And yeah, you got a good point, maybe better to leave the flags...

**Comment by Foxushka** [comment]
I've pushed English to our project https://crowdin.com/project/chameleon-ultra-gui, can you move your Spanish translation there?

**Pull request merged**